packages/keyring-internal-api/package.json

```diff
@@ -68,6 +68,9 @@
     "typedoc": "^0.25.13",
     "typescript": "~5.6.3"
   },
+  "peerDependencies": {
+    "@metamask/keyring-api": "workspace:^"
```

Comment:
To enforce the relationship with the `keyring-api` we make it now a peer dep, so the consumer has to provide the correct `keyring-api` version on its own.